src/test/fuzz/checkqueue.cpp

```diff
@@ -39,13 +39,13 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     const unsigned int batch_size = fuzzed_data_provider.ConsumeIntegralInRange<unsigned int>(0, 1024);
     CCheckQueue<DumbCheck> check_queue_1{batch_size};
     CCheckQueue<DumbCheck> check_queue_2{batch_size};
-    std::vector<DumbCheck> checks_1;
-    std::vector<DumbCheck> checks_2;
+    std::vector<DumbCheck*> checks_1;
+    std::vector<DumbCheck*> checks_2;
     const int size = fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 1024);
     for (int i = 0; i < size; ++i) {
         const bool result = fuzzed_data_provider.ConsumeBool();
-        checks_1.emplace_back(result);
-        checks_2.emplace_back(result);
+        checks_1.emplace_back(new DumbCheck(result));
```

Comment:
Is this leak unproblematic? How many times does this run? (Obviously if it runs once per execution of the test binary it's fine, but it looks like it may run repeatedly.)

If nothing else, at least comment explaining why the leak is okay. (I'm curious why the original code had to be changed in the first place.)

Reply:
Ah, it looks like this got fixed in a later commit.